README.md

```diff
@@ -1,26 +1,84 @@
 # fm-overlays
+
 A Fieldmanager extension to render dynamic content in modal boxes/overlays on a site.
 
+
 ## Install & Setup
 
 Requires [Fieldmanager](https://github.com/alleyinteractive/wordpress-fieldmanager) to be installed and activated.
 
-### Overlay Types
-fm-overlays currently supports two types of content; either a single **Image** or the contents of the **WYSIWYG editor**.
 
-### Overlay Priority
-Each overlay has a display priority.  Use the `menu_order` attribute to set an overlay's priority.  The larger the `menu_order` the higher the display priority.  If no priorities have been set then *post date* is used instead.
+
+### Types
+
+fm-overlays currently supports two types of content; either a single **Image** or Rich Text.
+
+#### Image
+
+Image overlays utilize both the `srcset` attribute and the js `window.resize` event to ensure both proper resolution and aspect ratios are maintained regardless of device.  When creating an image overlay the user can also add a custom `Image Link` URL to wrap the image with a `<a>` tag, the `target` of this link can be customized as well.
+
+#### Rich Text
+
+Rich Text overlays are created using the classic **WYSIWYG editor**.
+
+### Conditionals
+
+Overlay conditionals allow the user to target where on the site the overlay will display.  The following conditionals are currently supported:
+
+- is_home
+- is_front_page
+- *is_category*
```

Comment:
I was debating the best way to indicate that those conditions support specific targeting.  Not sure if changing to:

_is_category_ : supports category targeting

would be more understandable?